### PR TITLE
Fix XP-Pen G430S_B configuration

### PR DESCRIPTION
**Pull request by Vlothx**
Fixes the ActiveReportID, cursor was not moving without this fix.

### CODE REVIEW

File: TabletDriverLib/Configurations/XP-Pen/G430S_B.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+  "InputReportLength": 12,
+  "ReportParser": null,
+  "CustomInputReportLength": 0,
+  "CustomReportParser": null,
+  "Width": 101.6,
+  "Height": 76.2,
+  "MaxX": 10160.0,
+  "MaxY": 7620.0,
+  "MaxPressure": 8191,
+  "ActiveReportID": 12,
```
Comment (InfinityGhost) [inline TabletDriverLib/Configurations/XP-Pen/G430S_B.json:14]: Would be a good idea to verify if this is working among multiple users, also across both supported platforms

Comment (usernamemadewheniplayedminecraft) [reply TabletDriverLib/Configurations/XP-Pen/G430S_B.json:14]: On Arch Linux kernel v5.6.10-arch1-1 and Windows 10 LTSC v1809, I used the modified .json file from this PR. After using the modified .json file the tablet went from using full area to the area specified on OTD. Both drag and hover feel normal.

Comment (InfinityGhost) [reply TabletDriverLib/Configurations/XP-Pen/G430S_B.json:14]: If we can get a test on Windows we should be good to call this working

Comment (InfinityGhost) [reply TabletDriverLib/Configurations/XP-Pen/G430S_B.json:14]: Was just notified that it was tested on Windows by 2 people already and confirmed working. Going to merge.

**Pull request merged**